docker-compose.yml

```diff
@@ -37,15 +37,22 @@ services:
       REVA_LDAP_IDP: https://${OCIS_DOMAIN:-localhost}:9200
       OCIS_LOG_LEVEL: debug
       REVA_TRANSFER_EXPIRES: 86400
-      REVA_STORAGE_EOS_DRIVER: eoshome
-      REVA_STORAGE_EOS_DATA_DRIVER: eoshome
+      REVA_STORAGE_EOS_DRIVER: eos
+      REVA_STORAGE_EOS_DATA_DRIVER: eos
       REVA_STORAGE_EOS_NAMESPACE: "/eos/dockertest/reva/users"
       REVA_STORAGE_EOS_MASTER_URL: ${EOS_MGM_URL:-root://mgm-master.testnet:1094}
       REVA_STORAGE_EOS_SLAVE_URL: ${EOS_MGM_URL:-root://mgm-master.testnet:1094}
-      REVA_STORAGE_EOS_LAYOUT: "{{substr 0 1 .Id.OpaqueId}}"
+      REVA_STORAGE_EOS_LAYOUT: "{{substr 0 1 .Id.OpaqueId}}/{{.Id.OpaqueId}}"
       REVA_FRONTEND_URL: https://${OCIS_DOMAIN:-localhost}:9200
       REVA_DATAGATEWAY_URL: https://${OCIS_DOMAIN:-localhost}:9200/data
-
+      # TODO dav/files namespace is broken because it should use /home not /oc or /eos
```

Comment:
I've moved all those to https://github.com/owncloud/product/issues/129#issuecomment-680087440, please remove these lines